### DISCUSSION TITLE
参考文献名称大小写问题

### DISCUSSION THREAD

**Discussion by EdmundWhis** [stone-zeng/fduthesis#334] [Q&A]
参考文献的文献标题似乎被强制小写了，即使是应该大写的人名也被小写。
如何修复这一问题，或者让大小写与*.bib文件中的大小写保持一致？
<img width="499" alt="Snipaste_2024-03-28_20-41-02" src="https://github.com/stone-zeng/fduthesis/assets/42163501/f27134bf-195f-42dd-874d-61a25a6f0301">

<img width="775" alt="Snipaste_2024-03-28_20-40-35" src="https://github.com/stone-zeng/fduthesis/assets/42163501/07407925-5bb0-4de7-8963-81def67433e1">

**Comment by stone-zeng** [comment]
`...{Lie} groups`，或者整个包起来 `title = {{...}}`。

**Comment by JackChou1996** [comment]
> 参考文献的文献标题似乎被强制小写了，即使是应该大写的人名也被小写。 如何修复这一问题，或者让大小写与*.bib文件中的大小写保持一致？ <img alt="Snipaste_2024-03-28_20-41-02" width="499" src="https://private-user-images.githubusercontent.com/42163501/317728624-f27134bf-195f-42dd-874d-61a25a6f0301.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjYxMTc5MzgsIm5iZiI6MTc2NjExNzYzOCwicGF0aCI6Ii80MjE2MzUwMS8zMTc3Mjg2MjQtZjI3MTM0YmYtMTk1Zi00MmRkLTg3NGQtNjFhMjVhNmYwMzAxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTEyMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMjE5VDA0MTM1OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdiNTRlNGRiNzc5ZDJkY2JjYTQ1ZWYyMjE1NzMxZTUxYzQ4NGZjOTUwZTg4Yzk0NTEyMzQ5YzY3NDMwYTczYmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.V3vw1iJX-cfo1dAqwZoPgGaj5Ks6MBtjAm5Zpwr0aBU">
> 
> <img alt="Snipaste_2024-03-28_20-40-35" width="775" src="https://private-user-images.githubusercontent.com/42163501/317728608-07407925-5bb0-4de7-8963-81def67433e1.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjYxMTc5MzgsIm5iZiI6MTc2NjExNzYzOCwicGF0aCI6Ii80MjE2MzUwMS8zMTc3Mjg2MDgtMDc0MDc5MjUtNWJiMC00ZGU3LTg5NjMtODFkZWY2NzQzM2UxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTEyMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMjE5VDA0MTM1OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkzMTUxY2Y1Y2QxYTQ1ZDViMzliMjBmODg0ZWU3MGFjNTUyMjFmZTFhNjY4YmEzZWJiYmNhYTJlODczOGU2NjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.zM75IsT2lrm9XUEP12i4RPPiO_VVAgXMCFBFyPOHFzI">

参考文献英文名不是要小写的吗
